recipes/r-xmsannotator/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "xMSannotator" %}
+{% set version = "1.3.2" %}
+{% set github = "https://github.com/recetox/xmsannotator" %}
+
+package:
+  name: "r-{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "{{ github }}/archive/packaging.tar.gz"
+  sha256: 22ec4034585ecde641f531693acaab4145617f9b1cd7787eda0c71e4cda961ea
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/
+    - lib/R/lib/
+
+
+requirements:
+  host:
+    - r-base
+    - r-dplyr >=1
+    - r-flashclust
+    - r-wgcna
+    - r-tidyr
+    - r-purrr
+    - r-tibble
+  run:
+    - r-base
+    - r-dplyr >=1
+    - r-flashclust
+    - r-wgcna
+    - r-tidyr
+    - r-purrr
+    - r-tibble
+    - r-data.table
+    - bioconductor-rhdf5
+    - r-rcpp
+    - r-arrow
+
+test:
+  commands:
+    - $R -e "library('xmsannotator')"
+
+about:
+  home: "{{ github }}"
+  license: MIT
```

Comment:
The code is licensed under GPL2.0. See https://github.com/yufree/xMSannotator/blob/master/DESCRIPTION#L10